src/should-forward-property.js

```diff
@@ -600,17 +26,18 @@ const isCustomAttribute = RegExp.prototype.test.bind(
 )
 
 const hasItem = (list, name) => list.indexOf(name) !== -1
-const isHtmlProp = name => hasItem(htmlProps, name)
+const isHtmlProp = (name, tagName) => {
+  const elementAttributes = reactHTMLAttributes[tagName]
+
+  return hasItem([...globalReactHtmlProps, ...elementAttributes], name)
```

Comment:
Found one.

https://www.npmjs.com/package/fast-memoize

It claims to be 'fastest possible memoization library in JavaScript'. And it's 616B gzipped. 

If you want this to be implemented, maybe I can just open another branch and pr to add this?